nncf/tensorflow/graph/model_transformer.py

```diff
@@ -150,7 +152,7 @@ def _apply_transformation(self, transformation: TransformationCommand):
             raise TypeError('Transformation type {} does not support'
                             .format(transformation.type))
 
-    def _insert(self, target_point: TargetPoint, insertion_objects: List[Callable]):
+    def _insert(self, target_point: TargetPoint, insertion_objects: List[Callable], inserted_instance_idx: int):
```

Comment:
The list of insertion_objects and only one inserted_instance_idx are input for the '_insert' function. What do you think about the capability to determinate inserted_instance_idx for each insertion_objects?

Reply:
Changed to `insert_with_instance_indices: List[int]`